django/forms/models.py

```diff
@@ -1301,7 +1301,9 @@ def prepare_value(self, value):
         if (hasattr(value, '__iter__') and
                 not isinstance(value, str) and
                 not hasattr(value, '_meta')):
-            return [super(ModelMultipleChoiceField, self).prepare_value(v) for v in value]
+            # This fails within the list comp, so set it here.
+            s = super()
```

Comment:
Might want to the following to avoid the attribute lookup while you're at it.

```python
prepare_value = super().prepare_value
return [prepare_value(v) for v in value]
```